easybuild/easyconfigs/r/RAPSearch2/RAPSearch2-2.24.eb

```
@@ -0,0 +1,38 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au>
+# License::     GNU GPLv1
+#
+# Notes::
+##
+
+easyblock = 'MakeCp'
+
+name = 'RAPSearch2'
+version = '2.24'
+
+homepage = 'https://omics.informatics.indiana.edu/mg/RAPSearch2'
+description = """
+RAPSearch stands for Reduced Alphabet based Protein similarity Search
+"""
+
+toolchain = {'name': 'system', 'version': 'system'}
```

@robqiao this is normally just `SYSTEM`:
```suggestion
toolchain = SYSTEM
```

---

Wait, why was this moved to system level at all? We are definitely compiling stuff here, the previous use of GCC-9.3.0 was definitely the correct choice.

---

Hmm, distribution tarball includes a legacy `boost` which is only compatible with legacy gcc 4.8.5
I guess we can try to link a modern `boost` but I prefer to use the distributed code as much as possible. See what you think @Micket